dashboard/test/ui/features/learning_platform/teacher_dashboard/progress_standards_view.feature

```diff
@@ -42,9 +42,10 @@ Feature: Viewing and Printing Standards Progress
     And I wait until element "#teacher-dashboard" contains text "CSTA Standards in"
     And I wait until element "#uitest-standards-view" is visible
     And I wait until element "#uitest-progress-standards-table" is visible
-    And I wait until element "#uitest-progress-standards-table" contains text "Model daily processes by creating and following algorithms (sets of step-by-step instructions) to complete tasks."
-    And I wait until element "#uitest-progress-standards-table" contains text "Available in 1 Lesson:"
-    And I wait until element "#test-how-to-standards" contains text "How to use this information"
+    And element "#uitest-progress-standards-table" contains text "1A-AP-08"
+    And element "#uitest-progress-standards-table" contains text "Model daily processes by creating and following algorithms (sets of step-by-step instructions) to complete tasks."
+    And element "#uitest-progress-standards-table" contains text "Available in 1 Lesson:"
+    And element "#test-how-to-standards" contains text "How to use this information"
```

Comment:
these elements should already have been rendered by this point, so we do not need to perform an additional wait.